### DISCUSSION TITLE
Question about block weight

### DISCUSSION THREAD

**Discussion by zyd232** [kohya-ss/sd-scripts#469] [Q&A]
There are 17 blocks in a LoRA:
<html>
<body>

1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
BASE | IN01 | IN02 | IN04 | IN05 | IN07 | IN08 | MID | OUT03 | OUT04 | OUT05 | OUT06 | OUT07 | OUT08 | OUT09 | OUT10 | OUT11

</body>
</html>

But, there are 25 blocks to fill in the trainning scripts. And I found the following words in kohya's updates:
`No LoRA corresponds to the first block, but 25 blocks are specified for compatibility with 'LoRA block weight' etc. `

Someone please tell me the correspondence between the 17 blocks in LoRA and the 25 blocks in the training parameters.

**Comment by TingTingin** [comment]
locon's are a different type of lora and have more blocks the "compatibilty" is for them
